coalib/results/Diff.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, file_list=None,
         Creates an empty diff for the given file.
 
         :param file_list: The original (unmodified) file as a list of its
-                          lines.
+                          lines. It has to be provided if ``create`` is not set,
+                          else an error will be raised.
```

Comment:
ah could you also document the error with a `:raises ValueError:` tag or mention here explicitly `ValueError`?